pkg/api/pingpong_test.go

```diff
@@ -16,20 +16,21 @@ import (
 	"github.com/ethersphere/bee/pkg/jsonhttp/jsonhttptest"
 	"github.com/ethersphere/bee/pkg/p2p"
 	pingpongmock "github.com/ethersphere/bee/pkg/pingpong/mock"
+	"github.com/ethersphere/bee/pkg/swarm"
 )
 
 func TestPingpong(t *testing.T) {
 	rtt := time.Minute
-	peerID := "124762324"
-	unknownPeerID := "55555555"
+	peerID := "ca1e9f3938cc1425c6061b96ad9eb93e134dfe8734ad490164ef20af9d1cf59c"
```

Comment:
Use swarm.Address for peerID, unknownPeerID and errorPeerID, so that the check in pingpongmock just to use swarm.Address.Equal.